src/Microsoft.AspNet.Razor/Parser/CSharpCodeParser.Statements.cs

```diff
@@ -13,6 +13,9 @@ namespace Microsoft.AspNet.Razor.Parser
 {
     public partial class CSharpCodeParser
     {
+        private static readonly Func<CSharpSymbol, bool> IsValidStatementSpacingSymbol =
+            IsSpacingToken(includeNewLines: true, includeComments: true);
```

Comment:
It's the difference of invoking a method to retrieve the value vs. having it stored. Not worth changing.